yuhow/yuhowNote.tex

```diff
@@ -33,27 +82,156 @@
+  \textit{For $\alpha > 0$, this average is biased toward the one among $x_+$ and $x_-$ with the smaller magnitude. For the same value of $|x_+|$ and $|x_-|$, the bias increases as $\alpha$ increases. Thus, we should always choose $\alpha \geq 0$.}
+  After few hours work, I still couldn't understand the meaning of this description.
+  \item A description in the second-last paragraph of the right column in page 310 mentioned that:
+  \textit{$(u_{mx\pm})^n_j$ are constructed using only the data associated with the mesh points (j - 1/2, n - 1/2) and (j + 1/2, n - 1/2), the effect of this modification is highly local; i.e., it generally will not cause the smearing of shock discontinuities.}
+  I didn't quite understand this modification is highly 'local' and will not cause the smearing of shock discontinuities.
+ \end{enumerate}
 
-\section{Questions in section of the $\alpha-\mu$ scheme}
+\section{Formula derivation}
+ \label{sec:formula_derivation}
+Noting some important deriviations of equations listed in Prof. S. C. Chang's CESE paper~\cite{CESE_Shin_Chung_Chang_1995}
  \begin{enumerate}
-  \item Why does Eq. (2.4) imply Eq. (2.5). ?
-  \item Why can Eq. (2.9) be proved using the fact that the total flux of $h^{*}$ leaving the boundary of any space-time region that is the union of any combination of CEs vanishes? Can't figure it out.
-  \item What's the finite-difference appromixation?
-  \item What's the meaning of "the $\alpha-\mu$ scheme uses a mesh that is staggered in time"?
-  \item What's the Lax scheme?
-  \item What's the amplification factors? Also what's their meaning/usage in the Leapfrog/DuFort-Frankel scheme?
-  \item What's the meaning of "two-level" and "three-level" scheme?
-  \item Why does not solutions of Eq. (2.22) dissipate with time? Or why is "no dissipation" equivalent to "neutrally stable"?
-  \item Why the total flux leaving any conservation element is zero? This question is relevant to the definition of Eq. (2.28).
-  \item Why does the term "$-\mu\partial^{2}u^{*}(x, t; j, n)/\partial x^{2} $ vanish "in Eq. (2.29)?
-  \item (In page 303) Why is the condition that Eq. (2.30) being valid uniformly within an SE stronger than Eq. (2.28) for a higher-order expansion? Are they the same thing but just in different form?
-  \item Eq. (2.33) maybe is wrong. The partial derivative in the left-hand side of Eq. (2.33) should be with respect to x instead of t. Will try to double check with author though email. (STATUS: waiting for Prof. Chang's reply)
+  \item Derivation of Eq.(2.5) $(u_t)^n_j = -a(u_x)^n_j$:
+
+        \hspace{4mm}Substituting Eq.(2.3) into Eq.(2.1), then you can get Eq.(2.5).
+
+  \item Derivation of Eq.(2.12) $\frac{4}{(\Delta x)^{2}}F_{\pm}(j, n)=\pm(\frac{1}{2})[(1-\nu^{2}+\xi)(u_{x})^{n}_{j}+(1-\nu^{2}-\xi)(u_{x})^{n-1/2}_{j\pm1/2}]+\frac{2(1\mp\nu)}{\Delta x}(u^{n}_{j}-u^{n-1/2}_{j\pm1/2})$, where $\nu\equiv\frac{a\Delta t}{\Delta x}$ and $\xi\equiv\frac{4\mu\Delta t}{(\Delta x)^{2}}$:
```

Comment:
OK. I think your suggestion is very useful. I will follow this basic style rule.